charts/kubearchive/templates/database/ps-deployment.yaml

```diff
@@ -4,32 +4,32 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: {{ .Values.database.name | default "postgresql" }}
-  name: {{ .Values.database.name | default "postgresql" }}-deployment
-  namespace: {{ .Values.kubearchive.namespace }}
+    app: {{ tpl .Values.database.name . }}
+  name: {{ tpl .Values.database.name . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
I suggest removing this based on the fact that the `kubearchive` namespace is provided by the `-n` option in `helm install` command.
```suggestion
```

Reply:
removed the `namespace`